tests/pagination/tests.py

```diff
@@ -297,6 +298,12 @@ def test_get_page_empty_object_list_and_allow_empty_first_page_false(self):
         with self.assertRaises(EmptyPage):
             paginator.get_page(1)
 
+    def test_querysetpaginator_deprecation(self):
+        msg = 'QuerySetPaginator is deprecated in favor of Paginator.'
+        with self.assertWarnsMessage(RemovedInDjango31Warning, msg) as cm:
+            QuerySetPaginator([], 1)
+        self.assertEqual(cm.filename, __file__)
```

Comment:
What's the purpose of this line?

Reply:
I followed this pattern of testing deprecation warnings from other places in the tests. Sometimes it is there, other times not.

I believe it was intended to check that the stacklevel is correct so that the reported filename points to the site it was called from.

Reply:
Thanks, I forgot about that.